fsthttp/backend.go

```diff
@@ -0,0 +1,84 @@
+package fsthttp
+
+import (
+	"time"
+
+	"github.com/fastly/compute-sdk-go/internal/abi/fastly"
+)
+
+type TLSVersion uint32
+
+const (
+	TLSVersion1   TLSVersion = 0
+	TLSVersion1_1 TLSVersion = 1
+	TLSVersion1_2 TLSVersion = 2
+	TLSVersion1_3 TLSVersion = 3
+)
+
+type BackendOptions struct {
+	HostOverride        string
+	ConnectTimeout      time.Duration
+	FirstByteTimeout    time.Duration
+	BetweenBytesTimeout time.Duration
+	SSLMinVersion       TLSVersion
+	SSLMaxVersion       TLSVersion
+	CertHostname        string
+	CACert              string
+	Ciphers             string
```

Comment:
We should document the expected format for ciphers.  Are these colon-delimited OpenSSL-named ciphers?

(Docs for all of these would be good, but it's a must-have for `Ciphers`)